navbar-component.html

```diff
@@ -4,33 +4,45 @@
       <div class="navbar-header">
         <div br-slot="br-navbar-header">
           <div br-slot="br-navbar-header-inner"></div>
-          <button ng-if="$ctrl.getDisplayedMenus().length > 0" type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#br-navbar-collapse" aria-expanded="false">
+          <button ng-if="$ctrl.getDisplayedMenus().length > 0 && $ctrl.shouldDisplay('menu')"
+            type="button" class="navbar-toggle collapsed" aria-expanded="false"
+            ng-click="$ctrl.isNavCollapsed = !$ctrl.isNavCollapsed">
             <span class="sr-only">Toggle navigation</span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
             <span class="icon-bar"></span>
           </button>
-          <a class="navbar-brand" href="/">
-            <img ng-if="$ctrl.brand.src"
-              ng-src="{{$ctrl.brand.src}}"
-              width="{{$ctrl.brand.width}}"
-              height="{{$ctrl.brand.height}}"
-              alt="{{$ctrl.brand.alt}}" />
-            <span ng-if="!$ctrl.brand.src">{{$ctrl.siteTitle}}</span>
+          <a ng-if="$ctrl.shouldDisplay('brand')" class="navbar-brand" ng-href="{{$ctrl.service.site.url}}">
+            <img ng-if="$ctrl.service.site.brand.src"
+              ng-src="{{$ctrl.service.site.brand.src}}"
+              width="{{$ctrl.service.site.brand.width}}"
+              height="{{$ctrl.service.site.brand.height}}"
+              alt="{{$ctrl.service.site.brand.alt}}" />
+            <span ng-if="!$ctrl.brand.src">{{$ctrl.service.site.title}}</span>
+          </a>
+          <a ng-if="$ctrl.shouldDisplay('tenant') && !ctrl.isEmpty($ctrl.service.tenant)"
```

Comment:
This is good -- because it looks like, if `tenant` is empty, this won't display, which would be the common case.